### PR TITLE
Add a commonly expressed worry for Rust

### PR DESCRIPTION
**Pull request by joshtriplett**

### CODE REVIEW

File: surveys/2021-annual-survey/questions.md [hunk 1]
```diff
@@ -713,6 +713,7 @@ Select one:
 - Doesn't add a feature I want
 - Stagnation
 - Instability of the language
+- Development of the language not scalable/sustainable
```
Comment (rylev) [inline surveys/2021-annual-survey/questions.md:716]: Is this trying to capture the worry that the Rust project itself implodes or fails to keep up with the demands? I like this, but I'm not sure about the wording though I can't come up with a good alternative.

Comment (nrc) [inline surveys/2021-annual-survey/questions.md:716]:
```suggestion
- Development of the language is not scalable
```

I think having an answer here specifically about scaling is good. I think sustainability basically covers all of the answers, so this answer is better off without it.

Comment (joshtriplett) [inline surveys/2021-annual-survey/questions.md:716]:
```suggestion
- Development of the language not scalable
- Developers/maintainers of the language not paid/supported
```

Comment (nrc) [reply surveys/2021-annual-survey/questions.md:716]: lgtm

**Pull request closed without merging**